### DISCUSSION TITLE
Django Ninja?

### DISCUSSION THREAD

**Discussion by jensenbox** [reactive-python/reactpy-django#148] [Question] [answered]
Is there any interest in implementing Django Ninja?

https://django-ninja.rest-framework.com/

**Comment by rmorshea** [comment]
We have a Django integration called [reactpy-django](https://github.com/reactive-python/reactpy-django), but I'm not really sure how this Ninja framework would factor into that. @Archmonger, can you comment on this?

**Reply by Archmonger** [reply, marked as answer]
In theory, we have Django Ninja support out of the box. However, I have not tested this.

I have [created an issue](https://github.com/reactive-python/reactpy-django/issues/205) to add documentation for Django Ninja support.
